### ISSUE TITLE
[Bug Report] Performer scrape menu obstructs search results

### ISSUE THREAD

**Issue by Hakker** [stashapp/stash#1049]
**Describe the bug**
Scrape with when filled with plugins obstructs the search result dropdown

**To Reproduce**
Have actress plugin scrapers.
scrape with any scrapers that can be selected
fill in a partial name of the actor/actress so multiple results will show
result

**Expected behavior**
either move the scrape with button more to another side or the results a bit because now the scrape with popup will always overlap the results dropdown list obfuscating the last 3 results.

**Screenshots**
Sorry, on shitty 4g due ISP being a B****

**Stash Version: (from Settings -> About):**
0.40 release

**Desktop (please complete the following information):**
 - Win10 20H2
 - Firefox & Vivaldi(Chrome)
 - latest versions

**Smartphone (please complete the following information):**
not checked with phone will probably be the same or worse.

**Additional context**
This is purely a UI/UX issue this isn't a technical issue.

**Comment by Belleyy** [comment]
If your issue is this :
![image](https://user-images.githubusercontent.com/23707269/103936305-8b71d680-5127-11eb-9229-cbfeebf4be8c.png)

You can use this custom css 
```css
#scraper-popover {
    z-index: 1;
}

```

**Comment by hyde231** [comment]
> You can use this custom css atm (Settings > Interface > Custom CSS)
> 
> ```css
> #scraper-popover {
>     z-index: 1;
> }
> ```

This did **not** fix the issue on my Win10 FF 84.0.1 (64-Bit), stash version v0.4.0-36-g03a9d65
I had to do this one instead:
```css
.popover {
    .z-index: 2;
}
```
